packages/table/src/react/index.tsx

```diff
@@ -1,270 +1,196 @@
+  Row: typeof TableRow
+}
+type TableComponent = React.ForwardRefExoticComponent<TableProps> & TableStatics
+
+const Table = forwardRef<HTMLTableElement, TableProps>((props, ref) => {
+  const {
+    renderContainer: Container = defaultRenderContainer,
+    scrollable = true,
+    ...rest
+  } = props
   const themeName = useTheme()
 
-  const active =
-    typeof sort !== 'boolean' &&
-    typeof sort !== 'undefined' &&
-    vars.sorts[sort] &&
-    typeof onClick === 'function'
-
-  const style = props.style || {}
-  if (flex && !style.flex) style.flex = flex
-
-  const handleClick = onClick
-    ? (evt: React.MouseEvent) =>
-        onClick(evt, sort ? getToggledSort({ sort }) : undefined)
-    : undefined
-  const children = onClick ? (
-    <button {...styles.columnHeaderButton()} onClick={handleClick}>
-      <div {...styles.columnHeaderButtonInner(align)}>
-        {props.children}
-        {sort && getSortIcon({ sort })}
-      </div>
-    </button>
-  ) : (
-    <>
-      {props.children}
-      {sort && getSortIcon({ sort })}
-    </>
-  )
   return (
-    <div
-      role="columnheader"
-      {...styles.columnHeader({
-        themeName,
-        active,
-        align,
-        onClick: Boolean(onClick)
-      })}
-      {...props}
-      style={style}
-      {...(sort && {
-        'aria-sort':
-          sort === vars.sorts.desc
-            ? 'descending'
-            : sort === vars.sorts.asc
-            ? 'ascending'
-            : 'none'
-      })}
+    <Container
+      {...styles.container(themeName, { scrollable })}
+      {...(scrollable && { role: 'region', tabIndex: 0 })}
     >
-      {children}
-    </div>
+      <table ref={ref} role="table" {...styles.table(themeName)} {...rest} />
+    </Container>
   )
-}
+}) as TableComponent
 
-ColumnHeader.displayName = 'Table.ColumnHeader'
+const defaultRenderContainer: React.FC = props => <div {...props} />
 
-interface CellProps extends HTMLPropsFor<'div'> {
-  align?: ValueOf<typeof vars.aligns>
-  emphasis?: boolean
-  flex?: string
-}
-
-const Cell: React.FC<CellProps> = ({
-  flex,
-  emphasis = false,
-  align = vars.aligns.left,
-  ...props
-}) => {
-  const style = props.style || {}
-  if (flex && !style.flex) style.flex = flex
-
-  return (
-    <div
-      role="cell"
-      {...styles.cell(align, emphasis)}
-      {...props}
-      style={style}
-    />
-  )
-}
-
-Cell.displayName = 'Table.Cell'
+const TableBody = forwardRef<HTMLTableSectionElement, HTMLPropsFor<'tbody'>>(
+  (props, ref) => {
+    return <tbody ref={ref} {...props} />
+  }
+)
+TableBody.displayName = 'Table.Body'
 
-interface RowProps extends HTMLPropsFor<'div'> {
-  _tableHasDrawers?: boolean
+interface TableCellProps extends HTMLPropsFor<'td'> {
+  align?: ValueOf<typeof alignments>
 }
+const TableCell = forwardRef<HTMLTableCellElement, TableCellProps>(
+  (props, ref) => {
+    const { align = alignments.left, ...rest } = props
+    return <td ref={ref} {...styles.cell({ align })} {...rest} />
```

Comment:
We'll need to submit a new pr for the [docs search config](https://github.com/algolia/docsearch-configs/blob/master/configs/pluralsight_design-system.json) or add role="cell" here

Reply:
the new role is `gridcell`.

TODO: update docs search config